docs/self-managed/operational-guides/configure-multi-tenancy.md

```diff
@@ -5,50 +5,35 @@ sidebar_label: "Configure multi-tenancy"
 description: "Learn how to configure multi-tenancy in Camunda 8."
 ---
 
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
 :::caution
 Multi-tenancy is currently only available for Camunda 8 Self-Managed with authentication enabled [through Identity](/self-managed/identity/what-is-identity.md).
 :::
 
-To successfully configure multi-tenancy, you must do the following:
-
-- Ensure Identity is configured with a [database](/self-managed/identity/deployment/configuration-variables.md#database-configuration).
-- Enable the multi-tenancy flag globally through [Helm charts](/self-managed/setup/install.md) **or** via environment variables for each required component.
-
-Multi-tenancy must be enabled for each required component. Using the single global flag with Helm charts is recommended.
+To successfully configure multi-tenancy, you must enable the multi-tenancy flag through [Helm charts](/self-managed/setup/install.md)
+**or** via environment variables.
 
-## Helm charts
+<Tabs groupId="memberType" defaultValue="helm" queryString values={[{label: 'Helm Charts', value: 'helm', },{label: 'Environment Variables', value: 'environment', }]} >
+<TabItem value="helm">
 
 When using Helm charts, you can enable multi-tenancy globally with the flag `global.multitenancy.enabled`.
 Visit [the Helm chart configuration](https://artifacthub.io/packages/helm/camunda/camunda-platform#global-parameters) for additional details.
 
-## Environment variables
+</TabItem>
+<TabItem value="environment">
 
-Without Helm charts, multi-tenancy can be enabled using environment variables. This feature **must** be
-enabled in **all** required components, see:
+When using environment variables, you can enable multi-tenancy by setting the following variables:
 
-- [Identity](../../../self-managed/identity/deployment/configuration-variables/#feature-flags)
-- [Zeebe](../../../self-managed/zeebe-deployment/configuration/gateway-config/#zeebegatewaymultitenancy)
-- [Operate](../../../self-managed/operate-deployment/operate-configuration/#multi-tenancy)
-- [Tasklist](../../../self-managed/tasklist-deployment/tasklist-configuration/#multi-tenancy)
-- [Optimize]($optimize$/self-managed/optimize-deployment/configuration/multi-tenancy/)
-- [Connectors](../../../self-managed/connectors-deployment/connectors-configuration/#multi-tenancy)
```

Comment:
Should the multi-tenancy sections of these config docs be removed, or will there still be a need for it at the component level?

Reply:
We can always follow up later, but I did want to flag it for us to be aware of as other pages start needing cleanup with the new config.